src/core/session.ts

```diff
@@ -212,31 +210,31 @@ export class Session implements NavigatorDelegate {
   }
 
   notifyApplicationAfterClickingLinkToLocation(link: Element, location: Location) {
-    return dispatch("turbo:click", { target: link, detail: { url: location.absoluteURL }, cancelable: true })
+    return dispatch("turbo:click", { target: link, detail: { url: location.absoluteURL }, cancelable: true, bubbles: true })
```

Comment:
Since the only dispatchers are `document` or a `turbo-frame` for the majority of the events, would it make sense to limit `{ bubbles: true }` to:

* `turbo:click`
* `turbo:submit-start`
* `turbo:submit-end`

Reply:
Bubbling is something I'd really appreciate from a user point of view. It gives a lot of flexibility as to which part of the DOM I can hook some JS to respond to the events. Especially, it'd allow to just have a listener on `document` to handle all the `turbo-frame`s of the page. 

What would be the gain of not having them bubble?

In any case, I'd find it easier if all the events behaved the same regarding bubbling, saving developers the need to remember which ones do and which don't.